lib/smart_answer/calculators/property_fire_safety_payment_calculator.rb

```diff
@@ -0,0 +1,58 @@
+module SmartAnswer::Calculators
+  class PropertyFireSafetyPaymentCalculator
+    attr_accessor :year_of_purchase,
+                  :value_of_property,
+                  :live_in_london,
+                  :shared_ownership,
+                  :percentage_owned
+
+    EARLIEST_VALID_YEAR = 1945
+    OUTER_LONDON_VALUATION_LIMIT = 175_000
+    INNER_LONDON_VALUATION_LIMIT = 325_000
```

Comment:
Are these actually outer london and inner london, or are they outside london and inside london?

Reply:
Thanks Keith, you're right. I've changed them from OUTER/INNER to OUTSIDE/INSIDE